src/symbols/Edge/Edge.tsx

```diff
@@ -94,17 +109,18 @@ export const Edge: FC<Partial<EdgeProps>> = ({
   const [deleteHovered, setDeleteHovered] = useState<boolean>(false);
   const [center, setCenter] = useState<CenterCoords | null>(null);
   const { selections, readonly } = useCanvas();
-  const isActive = selections?.length
+  const isActive: boolean = selections?.length
     ? selections.includes(properties?.id)
-    : null;
+    : false;
 
+  // The "d" attribute defines a path to be drawn. See https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
```

Comment:
Happy to add this comment but its a pretty known thing.

Reply:
I didn't know it so I figure it'd be a good place to start for SVG beginners